djangocms_admin_style/sass/partials/_forms.scss

```diff
@@ -1,108 +1,251 @@
+}
+input[type="submit"],
+input[type="button"] {
+    @include button-common();
+    padding: 8px 10px !important;
+    &.default {
+        @include button-blue();
+        margin-top: 1px; 
+        padding: 7px 10px !important;
+    }
 }
 a.button {
-	text-decoration: none; cursor: pointer; @include button-common();
-	&.tab { border-radius: 0; }
+    @include button-common();
+    text-decoration: none;
+    cursor: pointer; 
+    &.tab {
+        border-radius: 0;
+    }
 }
 
 // date time
 .form-row {
-	p.datetime { line-height: 36px; }
+    p.datetime {
+        line-height: 36px;
+    }
 }
 
 // calendar
 div.calendar {
-	table {
-		caption { background-image: linear-gradient($background-color-dark, $background-color-medium);
-			background-color: $background-color-dark; color: $font-color-white !important; margin: 0;
-			border-top-radius: $default-border-radius; }
-		td { padding: 0; background: #f3f3f3; }
-		td a { padding: 6px 4px; }
-	}
-	.today a { background: $info-color-ultralight; }
+    table {
+        caption {
+            color: $font-color-white !important;
+            margin: 0;
+            background-color: $background-color-dark;
+            background-image: linear-gradient($background-color-dark, $background-color-medium);
+
+            border-top-radius: $default-border-radius;
+        }
+        td {
+            padding: 0;
+            background: #f3f3f3;
+        }
+        td a {
+            padding: 6px 4px;
+        }
+    }
+    .today a {
+        background: $info-color-ultralight;
+    }
 }
 div.calendarbox {
-	width: 200px !important;
-	table th, table td { line-height: 12px; }
-	.calendarnav-previous, .calendarnav-next {
-		background: none; color: white; text-decoration: none; padding: 4px 7px 6px;
-		&:hover, &:focus { background: #666; }
-	}
-	.calendar-shortcuts { background: transparent; padding: 10px 0 6px 0; border-color: #ddd; }
-}
-div.calendarbox, div.clockbox {
-	border-radius: $default-border-radius; background: #f3f3f3;
-	padding: 0; margin: 0; box-shadow: 0 1px 10px rgba(0,0,0,0.1); border: 1px solid #ccc;
-	caption, h2 { font-size: 14px; }
-	.calendar-cancel { border-bottom-radius: $default-border-radius; background: none; margin-top: 5px !important;
-		padding: 5px 0 !important; }
+    width: 200px !important;
+    table th,
+    table td {
+        line-height: 12px;
+    }
+    .calendarnav-previous,
+    .calendarnav-next {
+        color: white;
+        text-decoration: none;
+        padding: 4px 7px 6px;
+        background: none;
+        &:hover,
+        &:focus {
+            background: #666;
+        }
+    }
+    .calendar-shortcuts {
+        padding: 10px 0 6px 0;
+        border-color: #ddd; 
+        background: transparent;
+    }
+}
+div.calendarbox,
+div.clockbox {
+    margin: 0;
+    padding: 0;
+    border: 1px solid #ccc;
+    border-radius: $default-border-radius;
+    background: #f3f3f3;
+    box-shadow: 0 1px 10px rgba(0,0,0,0.1);
```

Comment:
I see csscomb doesn't add space between rgba values, maybe do this manually still? `rgba(0, 0, 0, 0.1)`